Week 4 Hangman/main.py

```diff
@@ -167,6 +170,11 @@ def play(cls) -> None:
         
         if not cls.wordGuessed:
             print("Out of Guesses")
+            print("The word was ",cls.word)
```

Comment:
word = RandomWordGenerator.generate()
---> can you identify what's the problem here ?

Reply:
No , can you please explain it to me